src/index/Index.cpp

```diff
@@ -205,6 +205,11 @@ void Index::setNumTriplesPerBatch(uint64_t numTriplesPerBatch) {
   return pimpl_->setNumTriplesPerBatch(numTriplesPerBatch);
 }
 
+// ____________________________________________________________________________
+const vector<uint64_t>& Index::getTextIndexIndices() const {
```

Comment:
It is easiest to make this return type an alias, then you don't have to change it so often, e.g.
`Index::TextIndexIndices`. (name should be also different, but this is for another time).